### ISSUE TITLE
why tensorflow model to IR the out put file is empty and there is anything wrong logs out 

### ISSUE THREAD

**Issue by cuicuizhang1989** [Microsoft/MMdnn#214]
~/anaconda2/bin/python -m mmdnn.conversion._script.convertToIR -f tensorflow -d teacher -n imagenet_teacher_cp.ckpt.meta --dstNodeName teacher_out -w imagenet_teacher_cp.ckpt
Parse file [imagenet_teacher_cp.ckpt.meta] with binary format successfully.
Tensorflow model file [imagenet_teacher_cp.ckpt.meta] loaded successfully.
Tensorflow checkpoint file [imagenet_teacher_cp.ckpt] loaded successfully. [16] variables loaded.
IR network structure is saved as [teacher.json].
IR network structure is saved as [teacher.pb].
IR weights are saved as [teacher.npy].
-rw-rw-r-- 1 cuicuizhang cuicuizhang        2  5月 26 16:24 teacher.json
-rw-rw-r-- 1 cuicuizhang cuicuizhang      212  5月 26 16:24 teacher.npy
-rw-rw-r-- 1 cuicuizhang cuicuizhang        0  5月 26 16:24 teacher.pb

the teacher model is create by myself is as follows:

 #coding=utf-8
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function

from tensorflow.contrib import layers
from tensorflow.contrib.framework.python.ops import arg_scope
from tensorflow.contrib.layers.python.layers import layers as layers_lib
from tensorflow.contrib.layers.python.layers import regularizers
from tensorflow.contrib.layers.python.layers import utils
from tensorflow.python.ops import array_ops
from tensorflow.python.ops import init_ops
from tensorflow.python.ops import nn_ops
from tensorflow.python.ops import variable_scope
import tensorflow as tf
import os
import numpy as np

def teacher_arg_scope(weight_decay=0.0005):
  """Defines the VGG arg scope.

  Args:
    weight_decay: The l2 regularization coefficient.

  Returns:
    An arg_scope.
  """
  with arg_scope(
      [layers.conv2d, layers_lib.fully_connected],
      activation_fn=nn_ops.relu,
      weights_regularizer=regularizers.l2_regularizer(weight_decay),
      biases_initializer=init_ops.zeros_initializer()):
    with arg_scope([layers.conv2d], padding='SAME') as arg_sc:
      return arg_sc

def conv2d(x, W, b, strides=1):
    # Conv2D wrapper, with bias and relu activation
    with tf.name_scope("teacherconv2d" ), tf.variable_scope("teacherconv2d" ):
      x = tf.nn.conv2d(x, W, strides=[1, strides, strides, 1], padding='SAME')
      x = tf.nn.bias_add(x, b)
      #            return tf.nn.relu(x)
      return x

def maxpool2d(x, k=2):
    # MaxPool2D wrapper
    with tf.name_scope("teachermaxpool2d") , tf.variable_scope("teachermaxpool2d" ):
      return tf.nn.max_pool(x, ksize=[1, k, k, 1], strides=[1, k, k, 1],
                            padding='SAME')

def batchnorm2d( x):
    fc_mean, fc_var = tf.nn.moments(
      x,
      axes=[0, 1, 2],  # 想要 normalize 的维度, [0] 代表 batch 维度
      # 如果是图像数据, 可以传入 [0, 1, 2], 相当于求[batch, height, width] 的均值/方差, 注意不要加入 channel 维度
    )
    scale = tf.Variable(tf.ones([x.shape[-1]]))
    shift = tf.Variable(tf.zeros([x.shape[-1]]))
    epsilon = 0.001
    x = tf.nn.batch_normalization(x, fc_mean, fc_var, shift, scale, epsilon)
    return x

def relu( x):
    return tf.nn.relu(x)

def teacher(  inputs,
  num_classes = 65,
  is_training = True,
  dropout_keep_prob = 0.5,
  spatial_squeeze = True,
  scope = 'teacher'):
    weights = {
    # 5x5 conv, 1 input, 32 outputs
    'wc1': tf.Variable(tf.random_normal([3, 3, 1, 32]), name="%s_%s" % ("teacher", "wc1")),
    # 5x5 conv, 32 inputs, 64 outputs
    'wc2': tf.Variable(tf.random_normal([3, 3, 32, 64]), name="%s_%s" % ("teacher", "wc2")),
    # 3x3 conv, 64 inputs, 128 outputs
    'wc3': tf.Variable(tf.random_normal([3, 3, 64, 128]), name="%s_%s" % ("teacher", "wc3")),
    # fully connected, 7*7*64 inputs, 1024 outputs
    'wd1': tf.Variable(tf.random_normal([8 * 4 * 128, 1024]), name="%s_%s" % ("teacher", "wd1")),
    # 1024 inputs, 10 outputs (class prediction)
    'out': tf.Variable(tf.random_normal([1024, num_classes]), name="%s_%s" % ("teacher", "out"))
    }

    biases = {
    'bc1': tf.Variable(tf.random_normal([32]), name="%s_%s" % ("teacher", "bc1")),
    'bc2': tf.Variable(tf.random_normal([64]), name="%s_%s" % ("teacher", "bc2")),
    'bc3': tf.Variable(tf.random_normal([128]), name="%s_%s" % ("teacher", "bc3")),
    'bd1': tf.Variable(tf.random_normal([1024]), name="%s_%s" % ("teacher", "bd1")),
    'out': tf.Variable(tf.random_normal([num_classes]), name="%s_%s" % ("teacher", "out"))
    }

    with tf.name_scope("teacherinputreshape"), tf.variable_scope(
                    "teacherinputreshape"):
      x = tf.reshape(inputs, shape=[-1, 30, 14, 1])

    # Convolution Layer
    with tf.name_scope("teacherconvmaxpool"), tf.variable_scope("teacherconvmaxpool"):
      conv1 = conv2d(x, weights['wc1'], biases['bc1'])
      conv1 = batchnorm2d(conv1)
      conv1 = relu(conv1)
      conv1 = maxpool2d(conv1, k=2)

      # Convolution Layer
      conv2 = conv2d(conv1, weights['wc2'], biases['bc2'])
      conv2 = batchnorm2d(conv2)
      conv2 = relu(conv2)
      conv2 = maxpool2d(conv2, k=2)

      conv3 = conv2d(conv2, weights['wc3'], biases['bc3'])
      conv3 = batchnorm2d(conv3)
      conv3 = relu(conv3)

#      conv3 = tf.layers.flatten(conv3)

    # Fully connected layer
    # Reshape conv2 output to fit fully connected layer input
    with tf.name_scope("teacherfclayer" ), tf.variable_scope("teacherfclayer" ):
      end_points_collection = 'teacher_out'
      fc1 = tf.reshape(conv3, [-1, weights['wd1'].get_shape().as_list()[0]])
      fc1 = tf.add(tf.matmul(fc1, weights['wd1']), biases['bd1'])
      fc1 = relu(fc1)
      # Apply Dropout
      #       fc1 = tf.nn.dropout(fc1, self.keep_prob)
      # Output, class prediction
      logits = tf.add(tf.matmul(fc1, weights['out']), biases['out']) / 5.0
      end_points = utils.convert_collection_to_dict(end_points_collection)
      if num_classes is not None:
        end_points['predictions'] = layers_lib.softmax(
          logits, scope='predictions')
      return logits, end_points
    return logits, end_points
